R/expect_length_linter.R

```diff
@@ -8,12 +8,12 @@
 #' @seealso [linters] for a complete list of linters available in lintr.
 #' @export
 expect_length_linter <- function() {
-  # TODO(michaelchirico): also catch expect_true(length(x) == 1)
-  xpath <- sprintf("//expr[
-    expr[1][SYMBOL_FUNCTION_CALL[text() = 'expect_equal' or text() = 'expect_identical']]
+  xpath <- glue::glue("//expr[
+    expr[1][SYMBOL_FUNCTION_CALL[text() = 'expect_equal' or text() = 'expect_identical' or text() = 'expect_true']]
     and expr[
       expr[1][SYMBOL_FUNCTION_CALL[text() = 'length']]
-      and (position() = 2 or preceding-sibling::expr[NUM_CONST])
+      and
+      position() = 2 or EQ or preceding-sibling::expr[NUM_CONST]
```

Comment:
this looks dangerous -- mixing `and` and `or` without parentheses is operator precedence hell :)

Reply:
Looks like I broke something while trying to fix this; will have a look tomorrow.